tests/phpunit/Mutator/Number/DecrementIntegerTest.php

```diff
@@ -476,6 +476,20 @@ public static function mutationsProvider(): iterable
                 PHP,
         ];
 
+        // see https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9524503 "Suppressing Unproductive Mutants"
+        yield 'It does not decrement when it is used in sleep functions' => [
+            <<<'PHP'
+                <?php
+                usleep(1_000);
+                sleep(1_000);
+                time_nanosleep(1_000, 0);
+                set_time_limit(1_000);
+
+                ini_set('memory_limit', 500); // in bytes
+                ini_set('memory_limit', -1);
```

Comment:
I would add more cases, besides valid escaped mutations here an obvious case missing IMO is if you have a legit numeric value in there that is not passed as a parameter of one of those functions be instead an element of an expression.

For instance:

```php
sleep((static fn () => $time === 1000 ? 10 : 100)());
```

quite an artificial case granted, but it illustrates that you have no guarantee that a numeric found within those functions are necessarily passed as a parameter to that function, in which case they should be mutated as usual.